changelogs/fragments/fix-build-files-manifest-walk.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

So I originally missed that this is not a user-facing bug that is triggered by using supported interfaces like API. Let's revert the fragment type to minor:
```suggestion
minor_changes:
```